src/proto/api_messages.js

```diff
@@ -72,6 +72,9 @@ const AnalyticsEvent = {
   ACTION_SWG_BUTTON_SELECT_OFFER_CLICK: 1018,
   ACTION_SWG_BUTTON_SHOW_CONTRIBUTIONS_CLICK: 1019,
   ACTION_SWG_BUTTON_SELECT_CONTRIBUTION_CLICK: 1020,
+  ACTION_USER_CONSENT_DEFERRED_ACCOUNT: 1021,
```

Comment:
This file seems to have a lot of unrelated changes. Could you make a separate PR to regenerate this file without the metering changes?